docs/multicluster/architecture.md

```diff
@@ -34,33 +36,40 @@ given ClusterSet.
 ## Antrea Multi-cluster Controller
 
-In a member cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for exporting resource to and importing resource from a leader
-cluster in a ClusterSet.
+Antrea Multi-cluster Controller watches ClusterSet, ClutserClaim, Service, ServiceExport etc.
+It has different responsibilities in member and leader clusters.
 
-In a leader cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for converting resources from different member clusters into one
-encapsulated resource as long as these resources have the same kind and match Namespace sameness.
+In a member cluster, Antrea Multi-cluster creates a Deployment with one replica that runs
+Antrea Multi-cluster Controller which is responsible for exporting resource to and importing
+resource from a leader cluster in a ClusterSet.
 
-In ClusterSet initialization, Antrea Multi-cluster Controller in a member cluster watches
+In a leader cluster, Antrea Multi-cluster creates a Deployment with one replica that runs
+Antrea Multi-cluster Controller which is responsible for converting resources from different
+member clusters into one encapsulated resource as long as these resources have the same kind
+and match Namespace sameness.
+
+When ClusterSet is initialized in member cluster, Antrea Multi-cluster Controller watches
 ClusterSet, ClusterClaim, and creates a MemberClusterAnnounce in the leader cluster.
 
-For exporting resources from the member cluster, it watches ServiceExport, Service and Endpoints
-resources, encapsulates Services and Endpoints into ResourceExports according to ServiceExports,
-and writes the ResourceExports to leader cluster. For resource importing, it watches ResourceImports
-from leader cluster, and creates multi-cluster Services and Endpoints with a prefix `antrea-mc-`
-plus exported Service name, and also ServiceImports which have the same name as the Service name.
+For resources exporting in the member cluster, Antrea Multi-cluster Controller watches ServiceExport,
+Service and Endpoints resources, encapsulates Services and Endpoints into ResourceExports
+according to ServiceExports, and writes the ResourceExports to leader cluster.
+
+For resource importing in the member cluster, Antrea Multi-cluster Controller watches ResourceImports
+from leader cluster, and creates multi-cluster Services and Endpoints with a prefix `antrea-mc-` plus
+exported Service name, and also ServiceImports which have the same name as the Service name.
 
-In a leader cluster, for ClusterSet initialization, Antrea Multi-cluster controller watches and
+When ClusterSet is initialized in leader cluster, Antrea Multi-cluster controller watches and
 validates the ClusterSet and Clusterclaim. For resource export/import, it watches ResourceExports
 and encapsulates them into ResourceImports.
 
-## Service Export and Import
+### Service Export and Import
```

Comment:
I meant adding a section at the same level of "## Antrea Multi-cluster Controller" called "## Multi-cluster Controller Service", so the sections are like:

```
## Antrea Multi-cluster Controller
## Multi-cluster Controller Service
### Service Export and Import
### Service Access Across Clusters
## Multi-cluster Gateway
### Multi-cluster Service Traffic Walk
## Antrea Multi-cluster NetworkPolicy
```

What you think? BTW, I removed "Antrea" from a few section titles. I feel no need to repeat it in all places.

Reply:
sure, I will refine it. thanks.

Reply:
I suppose you mean `## Multi-cluster Service` in the second item?

Reply:
Ah, yes

Reply:
Done